packages/rum-react/karma.conf.js

```diff
@@ -38,24 +38,22 @@ module.exports = function(config) {
         {
           test: /.jsx?$/,
           exclude: /node_modules/,
-          use: {
-            loader: 'babel-loader',
-            options: {
-              presets: [
-                [
-                  '@babel/preset-env',
-                  {
-                    targets: {
-                      ie: '11'
-                    },
-                    useBuiltIns: false,
-                    modules: 'umd'
-                  }
-                ],
-                ['@babel/preset-react']
+          loader: 'babel-loader',
+          options: {
+            presets: [
+              [
+                '@babel/preset-env',
+                {
+                  targets: {
+                    ie: '11'
+                  },
+                  useBuiltIns: false,
+                  modules: 'umd'
+                }
               ],
-              plugins: ['@babel/plugin-transform-destructuring']
-            }
+              ['@babel/preset-react']
+            ],
+            plugins: ['@babel/plugin-transform-destructuring']
```

Comment:
its not required, needed only when we use multiple loaders on same files. plus we push to existing plugins in coverage and it will break since we have different structure now. https://github.com/elastic/apm-agent-rum-js/blob/c743f701a1cd65104612461fd19f36faceba5ed4/dev-utils/karma.js#L154